content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -3,3 +3,51 @@ title: "Workload Resources"
 weight: 20
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+Ultimately, your applications run as containers inside
+{{< glossary_tooltip term_id="Pod" text="Pods" >}}; however, managing individual
+`Pods` would be a lot of effort. For example, if a `Pod` fails, you probably want to
+run a new `Pod` to replace it. Kubernetes can do that for you.
+
+You use Kubernetes API to create a workload
+{{< glossary_tooltip text="objects" term_id="object" >}} that represent a higher level
+abstraction than a `Pod`, and then the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manages
+`Pods` on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) and [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)
```

Comment:
I don't feel comfortable when talking about Deployment and ReplicaSet at the same abstraction layer, because they are NOT.